### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #1509: [DOCS] UPDATE README,SECURITY,AND CONTRIBUTING PAGE
### Issue Description

There are a lot of grammatical errors in the README, SECURITY,AND CONTRIBUTING PAGE

### Additional Context

_No response_

### Screenshots or Examples (if applicable)

_No response_

### Proposed Solution (optional)

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by abhisar-yadav**
Corrected grammatical errors in the Contributing page

## Fixes 
Closes #1509 
Relates #1553 and #1555

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -91,16 +91,16 @@ Thank you for taking the time to contribute to our project. Please take a moment
 
   - `name`: Name of the site
 
-  - `description`: A brief description of what the site does
-  - `url`: Add a valid URL of the site
-  - `category`: Choose an existing category or create a new one following the instructions mentioned above.
+  - `Description`: A brief description of what the site does
```
Comment (CBID2) [inline CONTRIBUTING.md:94]:
```suggestion
  - `description`: A brief description of what the site does
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -91,16 +91,16 @@ Thank you for taking the time to contribute to our project. Please take a moment
...
+  - `URL`: Add a valid URL of the site
+  - `Category`: Choose an existing category or create a new one following the instructions mentioned above.
```
Comment (CBID2) [inline CONTRIBUTING.md:96]:
```suggestion
  - `url`: Add a valid URL of the site
  - `category`: Choose an existing category or create a new one following the instructions mentioned above.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -91,16 +91,16 @@ Thank you for taking the time to contribute to our project. Please take a moment
...
   - `subcategory`: Choose an existing subcategory or create a new one following the instructions mentioned above.
   - `language`: This field is included for some of the resources, such as _React_ resources or _YouTube_ channel links. If multiple languages are involved, prioritize English if available. Otherwise, choose a language that you deem necessary.
 
 - You can use [JSONLint](https://jsonlint.com/) to check the correctness of the JSON to avoid failing tests during pull requests.
 
 **NOTE**
-When adding _YouTube_ channel link, please specify _the language_ of the channel they are using to teach for example English, Hindi, Spanish etc. In cases where the language is NOT specified, then just remove the `language` property.
+When adding _YouTube_ channel link, please specify _the language_ of the channel they are using to teach for example English, Hindi, Spanish, etc. In cases where the language is NOT specified, then just remove the `language` property.
```
Comment (CBID2) [inline CONTRIBUTING.md:103]:
```suggestion
When adding _YouTube_ channel links, please specify _the language_ of the channel they are using to teach for example English, Hindi, Spanish, etc. In cases where the language is not specified, then just remove the `language` property.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions. I kept the part where it describes the JSON format in lowercase because arrays are not capitalized

**Comment by abhisar-yadav** [comment]
kindly merge the PR 
THANK YOU

**Review by CBID2 — APPROVED** [review]
LGTM! You just need a second approval

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**